index.html

```diff
@@ -1800,7 +1801,7 @@ <h3>
             <ol>
               <li>
                 <a>Issue a developer warning</a> that the start URL is not
-                <a>within scope</a> of the navigation scope.
+                <a>within scope</a> of the scope URL.
```

Comment:
scope URL should either be a variable or a link here, no?

Reply:
Not really, because this is the pseudo-suggested text of the developer warning, not part of the algorithm itself. Same as how "the start URL" is already referred to here without variable or link.

It's kind of just suggesting that the user agent show a warning that reads "The start URL is not within scope of the scope URL". I even thought about removing the link to "within scope" for the same reason; this isn't a reference to the "within scope" concept. It's just suggested text.